build_tools/travis/after_success.sh

```diff
@@ -14,6 +14,12 @@ function setup_conda {
     source activate testenv
 }
 
+if [[ "$USE_COVERAGE" == "1" ]]; then
+    lcov --directory . --capture --output-file coverage.info # capture coverage info
+    lcov --remove coverage.info '/usr/*' --output-file coverage.info # filter out system
+    lcov --list coverage.info
+    bash <(curl -s https://codecov.io/bash)
+fi
```

Comment:
No I was looking for the compiler options. But I guess cmake handles them transparently.